internal/usage/usage.go

```diff
@@ -254,6 +254,7 @@ const (
 	SkipSampleData                  = "Don't load sample data into your Atlas Cluster."
 	ContainerRegion                 = "Atlas region where the container resides."
 	ContainerRegions                = "List of Atlas regions where the container resides."
+	Timestamp                       = "Timestamp in ISO 8601 date and time format in UTC when Ops Manager last reclaimed free space on a cluster’s disks."
```

Comment:
[nit] rename to prevent misuse and also a fix on copy but I would confirm with @mongodb/docs-platform 
```suggestion
	ReclaimFreeSpaceTimestamp                       = "The service reclaims the space after the current time passes the provided timestamp. If not set the current timestamp will be used."
```